tasks/section5.yml

```diff
@@ -418,17 +418,16 @@
       - rule_5.2.16
 
 - name: "SCORED | 5.3.1 | PATCH | Ensure password creation requirements are configured"
-  lineinfile:
+  ini_file:
```

Comment:
NAK on `ini_file` here since it's not actually an ini.  `ini_file` isn't currently part of ansible core, so we avoid it for now.

Reply:
no problem, reverted back to lineinfile